app/src/main/java/com/mealsmadeeasy/ui/home/glance/WeekAtAGlanceMealViewHolder.kt

```diff
@@ -25,4 +55,25 @@ class WeekAtAGlanceMealViewHolder(root: View) : RecyclerView.ViewHolder(root) {
         }
     }
 
+    private fun onMenuItemClick(menuItem: MenuItem): Boolean {
+        when (menuItem.itemId) {
+            R.id.menu_item_delete_meal -> {
+                val builder = AlertDialog.Builder(itemView.context!!)
+                builder.setPositiveButton(
+                        itemView.context.getString(R.string.week_at_a_glance_delete_prompt_positive_ack),  {_, _ ->
+                    onDeleteMeal(meal)
+                })
+                builder.setNegativeButton(
+                        itemView.context.getString(R.string.week_at_a_glance_delete_prompt_negative_ack), { _, _ ->
+                    // Do nothing.
+                })
+                builder.setMessage(itemView.context.getString(R.string.week_at_a_glance_delete_prompt))
+                builder.show()
```

Comment:
Because we give the user the option to undo this deletion, let's ask for forgiveness instead of permission. Instead of having a dialog to confirm this, just call `onDeleteMeal` and let someone else figure out what to do (i.e. delete the meal and offer to undo it)